hermann.gemspec

```diff
@@ -31,13 +31,9 @@ Gem::Specification.new do |s|
     # dependency of your gem. if it is a runtime dependencies the require_jars
     # file will be overwritten during installation.
     s.add_dependency 'jar-dependencies', '~>0.1.2'
-    s.requirements << "jar org.apache.kafka:kafka_2.10, 0.8.1.1"
-    s.requirements << "jar org.mod4j.org.eclipse.xtext:log4j, 1.2.15"
-    s.requirements << "jar org.scala-lang:scala-library, 2.10.1"
-    s.requirements << "jar com.yammer.metrics:metrics-core, 2.2.0"
-    s.requirements << "jar org.slf4j:slf4j-api, 1.7.2"
-    s.requirements << "jar log4j:log4j, 1.2.17"
-    s.requirements << "jar com.101tec:zkclient, 0.3"
+    s.requirements << "jar org.apache.kafka:kafka_2.10, ~>0.8.1.1"
+    s.requirements << "jar org.apache.curator:curator-framework, ~>2.7"
```

Comment:
you can add ```, :scope => :test``` to have it as test dependency: https://github.com/lookout/leafy/blob/master/leafy-metrics/leafy-metrics.gemspec#L21